src/lib/config.js

```diff
@@ -101,16 +137,37 @@ function validateLocalEnvConfig () {
   }
 }
 
+function validateNotificationEnv () {
+  // Validate notification signing public keys
+  const notifications = parseNotificationSignEnv()
+
+  if (notifications.must_verify) {
+    const ledgers = parseLedgers()
+    for (let ledgerObj of ledgers) {
+      const uri = ledgerObj.ledger
+      if (notifications.keys[uri] === undefined) {
+        throw new Error(`Missing notification signing keys for ledger: ${uri}`)
```

Comment:
I see some throw new Error().  Do all these Errors result in Connector not starting?  
Could you tell us the thoughts behind using throw new Error() as supposed to returning an error or throwing a more specific error?